tools/submit-pull-request

```diff
@@ -8,7 +8,7 @@
 #
 # Authors:
 # - Martin Barisits, <martin.barisits@cern.ch>, 2017
```

Comment:
Don't forget to add your name 😄

Reply:
For a small patch like this? If you say so :).

Reply:
BTW: If you enable Zenodo integration, then the author list will be compiled for you from the commits.